src/AppBundle/Controller/DefaultController.php

```diff
@@ -14,8 +14,8 @@ class DefaultController extends Controller
     public function indexAction(Request $request)
     {
         // replace this example code with whatever you need
-        return $this->render('default/index.html.twig', [
-            'base_dir' => realpath($this->getParameter('kernel.root_dir').'/..'),
-        ]);
+        return $this->render('default/index.html.twig', array(
```

Comment:
We replace old array syntax with a new one in https://github.com/symfony/symfony-standard/pull/898